### PR TITLE
pull-releases: Skip EOLed releases

### PR DESCRIPTION
**Pull request by mkutlak**
With configuration option SupportEOL you can select whether you want to
pull and in that way "support" EOLed releases.

With this option switched off EOLed releases won't be added to the database and
any incoming uReport from that release will be rejected as not supported.

### CODE REVIEW

File: src/pyfaf/actions/pull_releases.py [hunk 1]
```diff
@@ -83,6 +83,9 @@ def run(self, cmdline, db):
 
                 continue
 
+            if not opsys_plugin.eol and remote_status == "EOL":
```
Comment (ernestask) [inline src/pyfaf/actions/pull_releases.py]: Isn’t this dead code because of the `continue` above?

Comment (ernestask) [reply src/pyfaf/actions/pull_releases.py]: Oh, hurr, indentation.

Comment (ernestask) [inline src/pyfaf/actions/pull_releases.py]: What about logging something in this case?

**Comment by ernestask** [comment]
LGTM overall.

File: src/pyfaf/actions/pull_releases.py (hunk 1 above)
Comment (mkutlak) [reply src/pyfaf/actions/pull_releases.py]: debug is enough?

Comment (ernestask) [reply src/pyfaf/actions/pull_releases.py]: Yes, sir.

**Pull request merged**